packages/botonic-cli/src/util/processes.ts

```diff
@@ -0,0 +1,83 @@
+import { execSync, spawn } from 'child_process'
```

Comment:
I suggest
1) renaming this file to system-info.ts
2) moving sleep & spawnProcess to a new file called eg. system.ts